algo/src/main/java/com/hartwig/serve/sources/ckb/facility_curation/CkbFacilityCurationFilterFile.java

```diff
@@ -0,0 +1,50 @@
+package com.hartwig.serve.sources.ckb.facility_curation;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+
+import com.hartwig.serve.datamodel.serialization.util.SerializationUtil;
+
+import org.apache.commons.compress.utils.Lists;
+import org.jetbrains.annotations.NotNull;
+
+public class CkbFacilityCurationFilterFile {
```

Comment:
Class with only static functions should be `final` so that people can't derive from it.